### DISCUSSION TITLE
How to kill gradle daemon on exit?

### DISCUSSION THREAD

**Discussion by igorlfs** [mfussenegger/nvim-jdtls#399] [Q&A]
From my understanding, when working in a (Gradle) project, jdt.ls uses a Gradle daemon to handle LSP stuff, which is fine. The issue, however, is that when exiting neovim, said daemon remains open, **occupying a non-trivial amount of memory** (in my case, at least 600 MiB). Is this the expected behavior? If so, is there any way to disable it and have the daemon killed on exit?

**Comment by mfussenegger** [comment]
That's expected. See https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:stopping_an_existing_daemon
